### PR TITLE
Allow fine grain control over weights per cluster

### PR DESCRIPTION
**Pull request by danibachar**
Signed-off-by: I <danibachar89@gmail.com>

I update the annotation key, to supply full control of per cluster per service weight distribution

### CODE REVIEW

**Review by tpantelis — APPROVED** [review]

**Comment by dfarrell07** [comment]
Is this waiting on anything other than review from the relevant folks?

**Comment by tpantelis** [comment]
> Is this waiting on anything other than review from the relevant folks?

It needs another approval. @aswinsuryan @vthapar

**Review by vthapar — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**
